### PR TITLE
Fixed #32686 -- Removed unnecessary semicolon on collected multiline SQL for RunSQL.

### PR DESCRIPTION
**Pull request by hannseman**
https://code.djangoproject.com/ticket/32686

### CODE REVIEW

File: tests/migrations/test_operations.py [hunk 1]
```diff
@@ -3003,6 +3003,30 @@ def test_run_sql_noop(self):
             operation.database_forwards("test_runsql", editor, None, None)
             operation.database_backwards("test_runsql", editor, None, None)
 
+    def test_run_sql_adds_missing_semicolon_on_collect_sql(self):
+        project_state = self.set_up_test_model('test_runsql')
+        new_state = project_state.clone()
+        operation_with_semicolon = migrations.RunSQL(
+            '''
+            INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1);
+            ''',
```
Comment (pope1ni) [inline tests/migrations/test_operations.py:3012]: I guess, as we're relying on there being whitespace, we should make this explicit:
```suggestion
            'INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1);\n',
```

File: tests/migrations/test_operations.py [hunk 2]
```diff
@@ -3003,6 +3003,30 @@ def test_run_sql_noop(self):
...
+            migrations.RunPython.noop
+        )
+        operation_without_semicolon = migrations.RunSQL(
+            '''
+            INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1)
+            ''',
```
Comment (pope1ni) [inline tests/migrations/test_operations.py:3018]:
```suggestion
            'INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1)\n',
```

File: tests/migrations/test_operations.py [hunk 3]
```diff
@@ -3003,6 +3003,30 @@ def test_run_sql_noop(self):
...
             operation.database_backwards("test_runsql", editor, None, None)
 
+    def test_run_sql_adds_missing_semicolon_on_collect_sql(self):
+        project_state = self.set_up_test_model('test_runsql')
+        new_state = project_state.clone()
+        operation_with_semicolon = migrations.RunSQL(
+            '''
+            INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1);
+            ''',
+            migrations.RunPython.noop
```
Comment (pope1ni) [inline tests/migrations/test_operations.py:3013]: Trailing comma:
```suggestion
            migrations.RunPython.noop,
```

File: tests/migrations/test_operations.py [hunk 4]
```diff
@@ -3003,6 +3003,30 @@ def test_run_sql_noop(self):
...
+        )
+        operation_without_semicolon = migrations.RunSQL(
+            '''
+            INSERT INTO test_runsql_pony (pink, weight) VALUES (1, 1)
+            ''',
+            migrations.RunPython.noop
```
Comment (pope1ni) [inline tests/migrations/test_operations.py:3019]: Trailing comma:
```suggestion
            migrations.RunPython.noop,
```

**Review by pope1ni — CHANGES_REQUESTED** [review]
Hi @hannseman. Thanks for this - I've just got a few minor suggestions.

File: tests/migrations/test_operations.py (hunk 1 above)
Comment (hannseman) [reply tests/migrations/test_operations.py:3012]: Good point, thanks!

**Review by pope1ni — APPROVED** [review]
LGTM.

**Comment by felixxm** [comment]
@hannseman Thanks :+1: I pushed extra tests to a separate commit.

**Pull request merged**